### ISSUE TITLE
Sidebar application does not respect client's prefers-reduced-transparency setting

### ISSUE THREAD

**Issue by Fyorl** [foundryvtt/foundryvtt#13393]
Originally reported by citrea_lingua https://discord.com/channels/170995199584108546/1296292044011995136/1409434656008568832

-# Not *exactly* a bug so much as an accessibility concern, but was told this is the place to put it, as per this message: https://discord.com/channels/170995199584108546/1243227783082283140/1409275946846130230
**ALL MODULES DISABLED?** true
**OS, Hosting, Browser (if applicable)**: Linux on both ends (Ubuntu server-side, Mint client-side), accessed through Firefox
**Short Description of issue**:
- Play area can be seen through menus and menu buttons overlay play area rather than being on top of the menu; this makes it harder for some people (like me) to focus on the text, and subsequently impedes readability (and is just generally *extremely* irritating)
- Opacity setting has no effect on menu and popouts, and even at 1, affected elements are *not* fully opaque
- Chat tab does not have any background whatsoever, exasperating issue 1 and play elements can be accessed through the gaps between messages
- Accessibility settings for this don't exist, or are so hard to find I still have not found them even with help (these should probably be in interface configuration!)
**Simple steps to reproduce the issue**: Load any un-moduled foundry instance in dark mode - light mode has full opacity.
**Screenshots and/or console errors**:

<img width="1371" height="919" alt="Image" src="https://github.com/user-attachments/assets/5e600ce5-5143-43ba-be9a-b872b07fcf20" />
